index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="src/sass/main.scss" />
+    <script type="module" src="src/js/controller.js"></script>
```

Comment:
Should prefer using async or defer

https://stackoverflow.com/questions/436411/where-should-i-put-script-tags-in-html-markup/24070373#24070373

Reply:
`defer` is preferred in this project as the script depends on the DOM and should not be executed immediately after it downloads, which would happen with `async`.

Although, as javascript modules are being used, the `type=module` is set to `defer` by default so there is no need to explicitly write that.

Reference: https://webdeveloper.beehiiv.com/p/html-script-element-attributes-async-vs-defer-vs-typemodule

Reply:
Nice!